pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/CompactStats.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.common.policies.data;
+
+/**
+ * Statistics about compact.
+ */
+public interface CompactStats {
```

Comment:
Nit: I've been wondering if it is worth changing the name of these stats to `CompactionStats`. The documentation talks about `compaction`, and so I somewhat expect the metrics to have `compaction` in the metric's name. I can see why you chose `CompactStats`, but I also could see `CompactionStats` as a valid alternative. Do you have a strong opinion either way?

Reply:
Yes, I will change to CompactionStats.